packages/sdk-browser/src/storage.ts

```diff
@@ -4,12 +4,14 @@ import WindowWrapper from "./__polyfills__/window";
 export enum StorageKeys {
   AuthBundle = "@turnkey/auth_bundle", // LEGACY
   CurrentUser = "@turnkey/current_user",
```

Comment:
should we consider this deprecated?

Reply:
yeah should we remove completely? I don't see why that would cause any issues

Reply:
I'm down to just mark as // LEGACY or // DEPRECATED for now